x-pack/plugins/observability/public/pages/alerts/alerts_table_t_grid.tsx

```diff
@@ -370,7 +375,8 @@ export function AlertsTableTGrid(props: AlertsTableTGridProps) {
       filters: [],
       hasAlertsCrudPermissions,
       indexNames,
-      itemsPerPageOptions: [10, 25, 50],
+      itemsPerPage: 10,
```

Comment:
Might be nice to have a comment regarding why we only have `10`. If there's an issue covering the EUI update maybe reference that?

Reply:
I added a comment!